yb-voyager/cmd/exportSchema.go

```diff
@@ -122,8 +124,24 @@ func exportSchema() error {
 		} else if len(binaryCheckIssues) > 0 {
 			return fmt.Errorf("\n%s\n%s", color.RedString("\nMissing dependencies for export schema:"), strings.Join(binaryCheckIssues, "\n"))
 		}
+
+		source.DB().Disconnect() // connect again after assessment
+	}
+
+	err = runAssessMigrationCmdBeforExportSchemaIfRequired(cmd)
```

Comment:
there's a bunch of guardrails even below, I think we should we call it after all the guardrail checks below.

Reply:
Done